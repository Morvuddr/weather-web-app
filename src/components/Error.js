import React from 'react';

const Error = () => (
    <div className="Error">
        <h1>Не удалось загрузить данные</h1>
    </div>
);

export default Error;