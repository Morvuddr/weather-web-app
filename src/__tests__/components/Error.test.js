import Error from '../../components/Error';
import renderer from 'react-test-renderer';
import React from 'react';

describe('Error component', () => {
    test('matches the snapshot', () => {
        const tree = renderer.create(
            <Error />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});