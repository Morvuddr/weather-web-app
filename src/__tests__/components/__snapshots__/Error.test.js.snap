// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component matches the snapshot 1`] = `
<div
  className="Error"
>
  <h1>
    Не удалось загрузить данные
  </h1>
</div>
`;
